tests/frontend/pages/pools/pools_solution.py

```diff
@@ -0,0 +1,150 @@
+        super().__init__(self, *args, **kwargs)
+        self.driver = driver
+        self.endpoint = "/admin/#/solutions/pools"
+
+    def load(self):
+        url = urljoin(self.base_url, self.endpoint)
+        self.driver.get(url)
+
+    def create(self, wallet_name, name, cu=1, su=1, duration_unit="Day", time_to_live=1):
+        # switch driver to ifram and choose create option
+        self.wait("v-progress-linear__buffer")
+        iframe = self.driver.find_elements_by_tag_name("iframe")[0]
+        self.driver.switch_to_frame(iframe)
+        chat_box = self.driver.find_element_by_class_name("chat")
+        radios = chat_box.find_elements_by_class_name("v-radio")
+        create_radio = [radio for radio in radios if radio.text == "create"][0]
+        create_radio.click()
+        self.click_button("NEXT")
+
+        self.wait("v-progress-circular")
+
+        # set pool name in input element
+        name_element = chat_box.find_element_by_class_name("v-text-field__slot")
+        name_input = name_element.find_element_by_tag_name("input")
+        name_input.send_keys(name)
+
+        self.click_button("NEXT")
+
+        # set cu, su and time to live to input elements
+        pool_details = {
+            "Required Amount of Compute Unit (CU)": cu,
+            "Required Amount of Storage Unit (SU)": su,
+            "Please choose the duration unit": duration_unit,
+            "Please specify the pools time-to-live": time_to_live,
+        }
+        input_form = self.driver.find_element_by_class_name("v-form")
+        div = input_form.find_element_by_class_name("mb-4")
+        inputs_elements = div.find_elements_by_xpath("following-sibling::div")
+        for input_element in inputs_elements:
+            text = input_element.find_element_by_class_name("mb-4").text
+            if text in pool_details.keys():
+                input = input_element.find_element_by_tag_name("input")
+                input.send_keys(pool_details[text])
+
+        self.click_button("NEXT")
+
+        self.wait("v-progress-circular")
+
+        # select the Freefarm
+        farm_box = self.driver.find_element_by_class_name("v-select__selections")
+        farm_input = farm_box.find_element_by_tag_name("input")
+        farm_input.send_keys("Freefarm")
+
+        self.click_button("NEXT")
+        self.wait("v-progress-circular")
+
+        # select the wallet
+        chat_box = self.driver.find_element_by_class_name("chat")
+        wallets_box = chat_box.find_element_by_class_name("v-input__slot")
+        wallets = wallets_box.find_elements_by_class_name("v-radio")
+        wallet = [wallet for wallet in wallets if wallet.text == wallet_name][0]
+        wallet.click()
+
+        self.click_button("NEXT")
+        self.wait("v-progress-circular")
+        self.wait("progressbar")
+        self.click_button("FINISH")
+
+    def extend(self, wallet_name, name, cu=1, su=1, duration_unit="Day", time_to_live=1):
```

Comment:
`extend` and `create` methods are almost have the same code, can we get the common code in a helper method, then use it to create and extend?